package.json

```diff
@@ -3,11 +3,11 @@
   "version": "1.0.0",
   "private": true,
   "dependencies": {
-    "@ant-design/icons": "4.8.0",
+    "@ant-design/icons": "^5.0.1",
```

Comment:
```suggestion
    "@ant-design/icons": "5.0.1",
```

Reply:
Note that you need to do this manually and cannot use suggestion directly, otherwise yarn.lock will not be synchronized